python/ray/tests/test_cancel.py

```diff
@@ -258,5 +258,27 @@ def remote_wait(sg):
         ray.get(inner, timeout=10)
 
 
+@pytest.mark.parametrize("use_force", [True, False])
+def test_recursive_cancel(ray_start_regular, use_force):
+    @ray.remote
+    def inner():
+        while True:
+            time.sleep(0.1)
+
+    @ray.remote(num_cpus=1)
+    def outer():
+
+        x = [inner.remote()]
+        print(x)
+        while True:
+            time.sleep(0.1)
+
+    outer_fut = outer.remote()
+    outer_val = ray.get(outer_fut)
```

Comment:
how does this work? isn't `outer` running an infinite loop?

Reply:
This test needs love--I will fix with stephanie's suggestions of resources